templates/zerver/navbar.html

```diff
@@ -12,6 +12,7 @@
     <div class="column-middle-inner">
           <div id="streamlist-toggle" {%if embedded %} style="visibility: hidden"{% endif %}>
             <a href="#" id="streamlist-toggle-button" role="button"><i class="icon-vector-reorder"></i>
+              <span class="ariahide">Streamlist toggle button</span>
```

Comment:
Should this be using some more formal `ARIA` tag than a new CSS class?  E.g. `aria-label` or something?

Also, these strings, like our other user-facing strings, should be tagged for translation.